docs/tutorial.md

```diff
@@ -2,593 +2,253 @@
 
 **Table of Contents**
 
-- [Basics](#basics)
```

Comment:
Instead of removing the js part, can you move it to another file ?

Reply:
I feel like its better to make others learn on their own instead of putting it in the document. It's how to use Mineflayer, not how to use JavaScript.

Reply:
I don't think that makes sense. Most users are blocked by their absence of knowledge of JavaScript

Reply:
I have to agree with @MalkistCoder. Documentation should be exclusively relating to mineflayer. Full-fledged tutorials should be extracted to things like blog posts, or, if you really care for it, to a separate repository.

Reply:
Tutorials are here to help concrete users solve their issues. They are not made for theoretical users.
Most of our users don't know javascript.

Reply:
Most of the people that use mineflayer know how to code, so I feel like the JavaScript section is not really necessary.

If someone needs/wants to learn JavaScript, I added multiple resources such as JavaScript.info (which uses multiple languages) and Codecademy's JavaScript tutorial (interactive), so even if they don't know JavaScript they can learn it easily and quickly.

Reply:
Feel free to read room on discord to convince yourself otherwise
You may also check the issues section in this repository